change/@fluentui-react-2020-10-26-17-19-22-feat-isConformant-classname-tests.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

This can be none since it's just touching tests.
```suggestion
  "type": "none",
```